elephant/utils.py

```diff
@@ -0,0 +1,21 @@
+import quantities as pq
+
+
+def check_quantities(var, var_name):
```

Comment:
Is this module a general file to provide utilities for `elephant`? That should be indicated.